src/components/interaction-designer/block/BlockToolbar.vue

```diff
@@ -0,0 +1,177 @@
+<template>
```

Comment:
Woww! Thank you so much for pulling this out. @poojahpatil90 and I were just talking about being excited to refactor Block.vue  😹 🤩